task-maker-format/src/ioi/dag/mod.rs

```diff
@@ -229,7 +229,7 @@ mod tests {
         let file = File::new("input");
         let (mut eval, _recv) = EvaluationData::new(tmpdir.path());
         let out = validator
-            .validate_and_bind(&mut eval, 0, 0, file.uuid)
+            .validate_and_bind(&mut eval, 0, None, 0, file.uuid)
```

Comment:
I'd appreciate also a test where this parameter is used, together with a test where it's None.